### PR TITLE
add buy-plank menu entry swap

### PR DESCRIPTION
**Pull request by Digbigpig**
Added the left-click buy-plank option for the lumber yard. Addresses  #1930
planks

### CODE REVIEW

**Comment by Nightfirecat** [comment]
This is [a rejected feature](https://github.com/runelite/runelite/wiki/Rejected-or-Rolled-Back-Features#menu-entry-swapping):

> `Menu reorder in shop/artisan skill screens`: Other clients don't implement these swaps, and would be too strong for numerous activities (examples include "Buy-50" swaps, "Make All" swaps for smithing/crafting, etc.)

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java [hunk 1]
```diff
@@ -202,6 +202,16 @@ default boolean swapPrivate()
 		return false;
 	}
 
+	@ConfigItem(
+		keyName = "swapPlank",
+		name = "Planks",
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: Rename this to "Buy planks"

Comment (Digbigpig) [reply runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: Thanks, committed the change.

**Comment by melkypie** [comment]
This was added in menu swapper extended in the plugin hub

**Pull request closed without merging**